easybuild/easyconfigs/m/mdocfile/mdocfile-0.1.3-foss-2023a.eb

```
@@ -0,0 +1,33 @@
+# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2024/10
+easyblock = 'PythonBundle'
+
+name = 'mdocfile'
+version = '0.1.3'
+
+homepage = 'https://github.com/teamtomo/mdocfile'
+description = """SerialEM mdoc files as pandas dataframes."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
```

I think this can be dropped to `gfbf`
```suggestion
toolchain = {'name': 'gfbf', 'version': '2023a'}
```